styles.css

```diff
@@ -0,0 +1,17 @@
+.full-page {
+    color: white;
+    top: 50px;
+    position: relative;
+    opacity: 0.9;
+    
```

Comment:
In order to avoid linter errors i suggest you remove the space here and any other such spaces